docs/overview/install.rst

```diff
@@ -79,28 +57,40 @@ Your machine is now ready to install Ocean software.
 Install Ocean Software
 ======================
 
-The simplest way to start is to install `dwave-ocean-sdk <https://github.com/dwavesystems/dwave-ocean-sdk>`_
-for the full suite of Ocean tools.
+The simplest way to start is to install
+`dwave-ocean-sdk <https://github.com/dwavesystems/dwave-ocean-sdk>`_ for the full
+suite of Ocean tools.
+
+You can :code:`pip install` the SDK inside your newly created virtual environment.
 
-* You can :code:`pip install` the SDK inside your newly created virtual environment, typically
-  with a command such as this or similar:
+* For a standard installation of the latest released version from
+  the `Python Package Index (PyPI) <https://pypi.org/>`_ repository:
 
   .. code-block:: bash
 
-      pip install dwave-ocean-sdk
+     pip install dwave-ocean-sdk
 
-* Alternatively, you can clone `dwave-ocean-sdk <https://github.com/dwavesystems/dwave-ocean-sdk>`_ repo
-  and install the SDK to your virtual environment; for example:
+* Advanced users might directly install the latest (development) SDK from the
+  `D-Wave GitHub <https://github.com/dwavesystems/dwave-ocean-sdk>`_ repository
+  using a command such as,
 
   .. code-block:: bash
 
-      git clone https://github.com/dwavesystems/dwave-ocean-sdk.git
-      cd dwave-ocean-sdk
-      python setup.py install
+     pip install git+https://github.com/dwavesystems/dwave-ocean-sdk
+
+Alternatively, you can clone the
+`dwave-ocean-sdk <https://github.com/dwavesystems/dwave-ocean-sdk>`_ repo
+and install the SDK to your virtual environment; for example:
+
+.. code-block:: bash
+
+    git clone https://github.com/dwavesystems/dwave-ocean-sdk.git
```

Comment:
For completeness, we might show how to clone submodules. Either with `--recursive`/`--recurse-submodules` above, or with additional `git submodule update --init`. Yet, since we didn't have this until now, and plan to eventually replace submodules, no point in adding it now. :smile: